doc/source/programs/gdal_vector_geom_op_explode_collections.rst

```diff
@@ -0,0 +1,80 @@
+.. _gdal_vector_geom_op_explode_collections_subsubcommand:
+
+================================================================================
+"gdal vector geom-op explode-collections" sub-sub-command
+================================================================================
+
+.. versionadded:: 3.11
+
+.. only:: html
+
+    Explode geometries of type collection of a vector dataset.
+
+.. Index:: gdal vector geom-op explode-collections
+
+Synopsis
+--------
+
+.. program-output:: gdal vector geom-op explode-collections --help-doc
+
+Description
+-----------
+
+:program:`gdal vector geom-op explode-collections` produces one feature for
+each geometry in any kind of geometry collection.
+
+For example if a feature contains a geometry ``MULTIPOINT(1 2,3 4)``, two
+corresponding target features will be generated: one with a geometry
+``POINT(1 2)`` and another one with geometry ``POINT(3 4)``
+
+It can also be used as a step of :ref:`gdal_vector_pipeline_subcommand`.
+
+Standard options
+++++++++++++++++
+
+.. include:: gdal_options/of_vector.rst
+
+.. include:: gdal_options/co_vector.rst
+
+.. include:: gdal_options/overwrite.rst
+
+.. include:: gdal_options/active_layer.rst
+
+.. option:: --active-geometry <ACTIVE-GEOMETRY>
+
+    Set the active geometry field from its name. When it is specified, only the
+    specified geometry field will be subject to the processing. Other geometry
+    fields will be not modified.
+    If this option is not specified, all geometry fields will be subject to the
+    processing.
+    This option can be combined together with :option:`--active-layer`.
+
+.. option:: --geometry-type <GEOMETRY-TYPE>
+
+   Change the geometry type of exploded geometries to be one of
+   ``GEOMETRY``, ``POINT``, ``LINESTRING``, ``POLYGON``, ``CIRCULARSTRING``, ``COMPOUNDCURVE``,
+   ``CURVEPOLYGON``, ``POLYHEDRALSURFACE`` or ``TIN``.
+   ``Z``, ``M`` or ``ZM`` suffixes can be appended to the above values to
+   indicate the dimensionality.
+
+.. option:: --skip
```

Comment:
Maybe something more descriptive here? I thought of `--skip-invalid` but that could be confused with OGC validity. `--skip-on-type-mismatch` ?

Reply:
changed to --skip-on-type-mismatch